django/test/runner.py

```diff
@@ -625,6 +625,26 @@ def is_discoverable(label):
     return os.path.isdir(os.path.abspath(label))
 
 
+def reorder_postprocess(reordered_suite):
+    # To make the TransactionTestCase initialize their data properly,
+    # they should know if the next TransactionTestCase needs initial data
+    # migrations serialized in the connection.
+    # Initialize _next_serialized_rollback attribute depending on the
+    # serialized_rollback option present in the next test in the test suite.
+    for previous_test, next_test in zip(
+        reordered_suite._tests[:-1],
+        reordered_suite._tests[1:]
+    ):
+        next_serialized_rollback = getattr(next_test, 'serialized_rollback', None)
+
+        # If the next test has no serialized_rollback option, it means that we
+        # have reached the end of the TransactionTestCases.
```

Comment:
Can this comment be wrapped up with the one above and made more succinct?
Possibly make that comment a docstring for this helper function too?